reference-implementation/lib/writable-stream.js

```diff
@@ -163,13 +163,37 @@ function WritableStreamAbort(stream, reason) {
   return promise;
 }
 
-function WritableStreamFinishAbort(stream) {
-  const error = new TypeError('Aborted');
-
+function WritableStreamError(stream, error) {
   stream._state = 'errored';
   stream._storedError = error;
 
-  WritableStreamRejectPromisesInReactionToError(stream);
+  // TODO: layering violation?
+  ResetQueue(stream._writableStreamController);
```

Comment:
Well right now WritableStreamDefaultControllerError does something rather different. I guess there is some collision in conventions for naming abtract ops in this area.

I think it is probably OK to leave as-is for now?